src/lib/containers/FacetsExperimental.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import * as SynapseConstants from 'lib/utils/SynapseConstants';
+
+export default class FacetsExperimental extends React.Component {
```

Comment:
where does FacetsExperimental fit in?  I think I understand QueryWrapper, Facets, StackedRowHomebrew, SynapseTable.